open-metadata-implementation/access-services/analytics-modeling/analytics-modeling-server/src/main/java/org/odpi/openmetadata/accessservices/analyticsmodeling/contentmanager/OMEntityDao.java

```diff
@@ -0,0 +1,251 @@
+import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.repositoryconnector.OMRSRepositoryHelper;
+import org.odpi.openmetadata.repositoryservices.ffdc.exception.EntityNotKnownException;
+import org.odpi.openmetadata.repositoryservices.ffdc.exception.EntityProxyOnlyException;
+import org.odpi.openmetadata.repositoryservices.ffdc.exception.FunctionNotSupportedException;
+import org.odpi.openmetadata.repositoryservices.ffdc.exception.InvalidParameterException;
+import org.odpi.openmetadata.repositoryservices.ffdc.exception.PagingErrorException;
+import org.odpi.openmetadata.repositoryservices.ffdc.exception.PropertyErrorException;
+import org.odpi.openmetadata.repositoryservices.ffdc.exception.RepositoryErrorException;
+import org.odpi.openmetadata.repositoryservices.ffdc.exception.TypeErrorException;
+import org.odpi.openmetadata.repositoryservices.ffdc.exception.UserNotAuthorizedException;
+
+/**
+ * The class implements access wrapper to retrieve data OMRS.
+ * 
+ */
+public class OMEntityDao {
+
+    protected final OMRSRepositoryConnector enterpriseConnector;
+    protected List<String> supportedZones;
+    private final AuditLog auditLog;
+    
+    private String context;
+    
+	private static final List<InstanceStatus> FILTER_ACTIVE = Arrays.asList(InstanceStatus.ACTIVE);
+
+    public OMEntityDao(OMRSRepositoryConnector enterpriseConnector, List<String> supportedZones, AuditLog auditLog) {
+        this.enterpriseConnector = enterpriseConnector;
+        this.auditLog = auditLog;
+        this.supportedZones = supportedZones;
+    }
+
+    /**
+     * Set context of the execution for logging.
+     * @param value high level method/operation requested by user.
+     */
+    public void setContext(String value) {
+    	context = value;
+    }
+    
+    /**
+     * Find entities matching criteria.
+     * @param matchProperties filter.
+     * @param typeName of the requested entity.
+     * @param fromElement index of the first entity.
+     * @param pageSize number of entities to return.
+     * @return list of matching entities
+     * @throws AnalyticsModelingCheckedException in case repository fails.
+     */
+    public List<EntityDetail> findEntities(InstanceProperties matchProperties, String typeName, int fromElement, int pageSize) throws AnalyticsModelingCheckedException {
+        // GDW the matchProperties passed to this method should have already converted any exact match string
+        // using the getExactMatchRegex repository helper method
+        OMRSRepositoryHelper repositoryHelper = enterpriseConnector.getRepositoryHelper();
+        TypeDef typeDef = repositoryHelper.getTypeDefByName(Constants.ANALYTICS_MODELING_USER_ID, typeName);
+        List<EntityDetail> existingEntities;
+        try {
+            auditLog.logMessage(context, AnalyticsModelingAuditCode.FIND_ENTITIES.getMessageDefinition(typeDef.getName(),  matchProperties.toString()));
```

Comment:
This audit log entry is an INFO not DEBUG - this means it should not be putput one each requests.  Either use SLF4J or turn the Category to be DEBUG